liquibase-standard/src/main/java/liquibase/sqlgenerator/core/InsertGenerator.java

```diff
@@ -67,8 +68,10 @@ public void generateValues(StringBuilder sql,InsertStatement statement, Database
 
         for (String column : statement.getColumnValues().keySet()) {
             Object newValue = statement.getColumnValues().get(column);
-            if ((newValue == null) || "NULL".equalsIgnoreCase(newValue.toString())) {
+            if ((newValue == null) || newValue.toString() == null) {
```

Comment:
If I'm not wrong, `newValue.toString() == null` could lead us to an NPE because newValue is not a `Null` instance, so if its value is `null`, the `toString()` call will cause an exception here. Shouldn't we update this line to:

```java 
if ((newValue == null) || new Null().equals(newValue)) 
```